### ISSUE TITLE
The question about how to train the contactnet and scorenet?

### ISSUE THREAD

**Issue by andyoung009** [mahaoxiang822/Generalizing-Grasp#3]
Thanks for your excellent and impressive work. I have 2 questions.
The readme file says "For contact-score joint optimization, first download the pretrained [contactnet](https://drive.google.com/file/d/1yMZ5rgloo0xbYvuR46t3sSKMvaVpOavx/view?usp=sharing) & [scorenet](https://drive.google.com/file/d/1didqsuweIbWb6UhL15IMhs2HrDhvC3EQ/view?usp=sharing) ", but I also find the loss.backward() in the optimization.py. So is there finetune in the optimization process? And how to train these two nets from scratch？
